src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,48 @@
-export const Tabs = () => {};
+import classNames from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+}
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const isActive = selectedTab.id === tab.id;
+
+            return (
+              <li
+                key={tab.id}
+                className={classNames({ 'is-active': isActive })}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  onClick={() => isActive || onTabSelected(tab)}
```

Comment:
Also, would be better make a separate function for it
```suggestion
                  onClick={() => {
                    if (!isActive) {
                      onTabSelected(tab);
                    }
                  }}
```